database/migrations.php

```diff
@@ -35,7 +35,16 @@ function migate(mysqli $db)
     CREATE TABLE Items
     (
     id INTEGER AUTO_INCREMENT PRIMARY KEY,
-    username VARCHAR(40) NOT NULL
+    itemName VARCHAR(50) NOT NULL,
+    description VARCHAR(255) NOT NULL,
+    sellerID INT NOT NULL,
```

Comment:
Very minor but just so I dont change your code, this should be sellerId (as referenced below) - it might error out otherwise